developer-docs/macos.md

```diff
@@ -43,13 +43,12 @@ Let's get started! Its time to put `brew` to work! Just copy paste commands into
 
 ```
 brew install pkg-config
-brew install scipy
 brew install libjpeg-turbo
 brew install libusb
 brew install portaudio
 # opencv will install ffmpeg, numpy, and opencv-contributions automatically
 # tbb is included by default with https://github.com/Homebrew/homebrew-core/pull/20101
-brew install opencv
+brew install opencv@3
```

Comment:
I would prefer updating the code base to opencv4. Please remove this change as well as the `aside` below.
```suggestion
brew install opencv
```